README.markdown

```diff
@@ -51,6 +51,17 @@ http {
     init_worker_by_lua_block {
         local hc = require "resty.upstream.healthcheck"
 
+        -- this should be defined in a separate module to avoid a global function definition
+        func_node_down = function(peer)
+            ngx.log(ngx.WARN, peer.name .. " is marked down")
+
+            -- do something else fancy with data about the peer, like notify a remote service
+        end
+
+        func_node_up = function(peer)
+            ngx.log(ngx.WARN, peer.name .. " is marked up")
```

Comment:
Maybe better to show good practices and pass 2 arguments instead of concatenating on the Lua land?
```
ngx.log(ngx.WARN, peer.name, " is marked up")
```